dolphinscheduler-ui/src/views/projects/task/constants/task-type.ts

```diff
@@ -47,6 +48,9 @@ export type TaskType =
 export type TaskExecuteType = 'STREAM' | 'BATCH'
 
 export const TASK_TYPES_MAP = {
+  JAVA: {
+    alias: 'JAVA'
+  },
```

Comment:
Does it have a document, if not, you need to set helperLinkDisable to true.

Reply:
Yes